utils/hct/HashStability.py

```diff
@@ -0,0 +1,142 @@
+
+import os
+import subprocess
+import platform
+import filecmp
+import argparse
+import shutil
+
+IS_WINDOWS = platform.system() == 'Windows'
+
+# Don't use close_fds on Windows.
+USE_CLOSE_FDS = not IS_WINDOWS
```

Comment:
Seems like this could be simpler as: `close_fds =  platform.system() != 'Windows'`

`IS_WINDOWS` is only used in one place, so we shouldn't really need to make it a variable.

Also `USE_CLOSE_FDS` doesn't really make sense as a name, `close_fds` reads more cleanly and matches the code style elsewhere.

Reply:
Used is_windows for later add .exe to dxa name.
Updated to close_fds.